doc/adr/0006-explicit-resources-for-declaring-service-bindings.md

```diff
@@ -0,0 +1,61 @@
+# 6. Explicit resources for declaring service bindings
+
+Date: 2024-01-16
+
+## Status
+
+Proposed
+
+Enables [7. Deprecate ServiceSync](0007-deprecate-servicesync.md)
+
+## Context
+
```

Comment:
We decided that the user must explicitly create listeners  on any sites that require an exposed service, accepting that this is an extra step for users but overall it gives users more control.

Reply:
That is really a consequence of the decision to remove service-sync which is described by adr 7 in which this point is now made.